src/commands/session.ts

```
@@ -21,12 +21,15 @@ import { handleSessionIdLookup } from './_session_id.ts';
 
 // All --id logic moved to ./_session_id.ts
 
+// All shared args except --order (not supported for session)
+const { order, ...sharedArgs } = sharedCommandConfig.args;
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)